fiware-region-sanity-tests/commons/neutron_operations.py

```diff
@@ -264,12 +264,12 @@ def find_networks(self, **kwargs):
         """
         found = []
         search = [(key.replace('router_', 'router:'), value) for (key, value) in kwargs.items()]
-        network_list = self.client.list_networks(retrieve_all=True).get('networks')
-        for network in network_list:
+        list = self.client.list_networks(retrieve_all=True).get('networks')
```

Comment:
I should not change the name of the variables in this case. I think that the semantic of the previos variables is better that the new ones.